test/test_results.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+import sys
+import os
+import subprocess
+import warnings
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from test.utils.custom_tsv_parser import ParseTSVFile
+
+PREF_CUTOFF = -4
+
+
+def get_scode_cscode_id(data):
+    # Create an id with scode and center with sorted values of scode and center
+    school_centers = []
+    for row in data:
+        scode = int(row["scode"])
+        center = int(row["cscode"])
+        sccode_center_id = sorted((scode, center))
+        sccode_center_id = "_".join(map(str, sccode_center_id))
+        school_centers.append(sccode_center_id)
+    return school_centers
+
+
+class TestSchoolCenter(unittest.TestCase):
```

Comment:
Project intends to use pytest, but for some reason these tests are written in unittest 🤔

Reply:
I can convert it to pytest later on but pytest should be able to run unittests as well with no issues

Reply:
Yes, it would be great to use pytest as it brings a lot of power to fixtures management and testing.